src/coreclr/vm/peimagelayout.cpp

```diff
@@ -1190,7 +1190,7 @@ NativeImageLayout::NativeImageLayout(LPCWSTR fullPath)
     PVOID loadedImage;
 #if TARGET_UNIX
     {
-        ErrorModeHolder mode(SEM_NOOPENFILEERRORBOX|SEM_FAILCRITICALERRORS);
+        ErrorModeHolder mode{};
```

Comment:
This is under `#if TARGET_UNIX` so the `ErrorModeHolder` is not really needed here.